clouddq/lib.py

```diff
@@ -195,6 +202,26 @@ def prepare_configs_from_rule_binding_id(
         {"configs_hashsum": sha256_digest(json.dumps(resolved_rule_binding_configs))}
     )
     configs.update({"progress_watermark": progress_watermark})
+    print("Incremental time fileter column")
```

Comment:
Remove these "print" lines? (I assume it's for debugging)

Reply:
yes, modified it to logger.